python-sdk/src/langwatch/experiment/experiment.py

```
@@ -399,6 +400,184 @@ def _display_df(df: pd.DataFrame) -> None:
+            experiment.print_summary()
+        """
+        from langwatch.experiment.platform_run import _is_notebook, _print_summary
+
+        result = self._build_run_result()
+        _print_summary(result)
+
+        should_exit = exit_on_failure if exit_on_failure is not None else not _is_notebook()
+        if should_exit and result.failed > 0:
+            sys.exit(1)
+
+    def _build_run_result(self) -> "ExperimentRunResult":
```

Ruff (0.15.10)

[error] Undefined name `ExperimentRunResult`

(F821)